### PR TITLE
Port code to Monocle 3.0.0-RC6

### PR DESCRIPTION
**Pull request by cquiroz**
Signed-off-by: Carlos Quiroz <carlos.m.quiroz@gmail.com>

### CODE REVIEW

File: modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala [hunk 1]
```diff
@@ -36,15 +36,15 @@ final case class SplitMono[A, B](get: A => B, reverseGet: B => A) {
...
     SplitMono(get.andThen(f.get), reverseGet.compose(f.reverseGet))
 
   /** Compose with another SplitEpi. */
-  def composeSplitEpi[C](f: SplitEpi[B, C]): Wedge[A, C] =
+  def andThen[C](f: SplitEpi[B, C]): Wedge[A, C] =
     Wedge(get.andThen(f.get), reverseGet.compose(f.reverseGet))
 
   /** Compose with an Iso. */
-  def composeIso[C](f: Iso[B, C]): SplitMono[A, C] =
+  def andTen[C](f: Iso[B, C]): SplitMono[A, C] =
```
Comment (tpolecat) [inline modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala:47]: Typo here. I guess we don't have a test for this ;-)

**Review by tpolecat — CHANGES_REQUESTED** [review]
Looks great. One typo.

File: modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala (hunk 1 above)
Comment (toddburnside) [inline modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala:47]: Typo

Comment (swalker2m) [reply modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala:47]: Not a typo, just spoken with an accent.

Comment (toddburnside) [reply modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala:47]: Rob beat me to the draw!

Comment (cquiroz) [reply modules/core/shared/src/main/scala/lucuma/core/optics/SplitMono.scala:47]: Darn

**Review by tpolecat — APPROVED** [review]

**Review by toddburnside — APPROVED** [review]
👍

**Comment by cquiroz** [comment]
We may want to wait for a final monocle before merging? This needs to propagate to all dependees.

**Review by toddburnside — APPROVED** [review]
LGTM

**Pull request merged**